<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pic_Show</class>
 <widget class="QWidget" name="Pic_Show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pic1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pic6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>560</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="time2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="time3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="time4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="time5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>620</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="time6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>620</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>520</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QLabel" name="hot6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>520</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
